Views/Products/EditInfo.cshtml

```diff
@@ -0,0 +1,59 @@
+@model Bangazon.Models.Product
+
+
+
+@{
+    ViewData["Title"] = "Product Edit";
+}
+
+<form asp-action="Edit" asp-route-id="@Model.ProductId">
+    <div class="form-horizontal">
+        <h4>Edit</h4>
+        <hr>
+        <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+        <div>
+            <label asp-for="Name" class="col-md-2 control-label">Product Name</label>
+            <div class="col-md-10">
+            <input asp-for="Name" class="form-control" placeholder="@Html.DisplayFor(model => model.Name)" />
+            <span asp-validation-for="Name" class="text-danger" />
+            </div>
+        </div>
+
+        <div>
+            <label asp-for="Price" class="col-md-2 control-label">Price</label>
+            <div class="col-md-10">
+            <input asp-for="Price"  class="form-control" placeholder="@Html.DisplayFor(model => model.Price)" />
+            <span asp-validation-for="Price" class="text-danger" />
+            </div>
+        </div>
+
+        <div>
+            <label asp-for="Description" class="col-md-2 control-label">Description</label>
+            <div class="col-md-10">
+            <input asp-for="Description" class="form-control" placeholder="@Html.DisplayFor(model => model.Description)" />
+            <span asp-validation-for="Description" class="text-danger" />
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label asp-validation-for="ProductTypeId" class="col-md-2 control-label"></label>
```

Comment:
Could we give this a label?
`<label asp-validation-for="ProductTypeId" class="col-md-2 control-label">Product Type</label>`